app/renderer/actions/Stub.js

```diff
@@ -0,0 +1,8 @@
+export const NAME_OF_ACTION_1 = 'NAME_OF_ACTION_1';
```

Comment:
is this file needed?

Reply:
These are 'Stub' files for future usage for copying and pasting to create new actions/reducers/components

Reply:
oh i see so independently of this PR you want to have stub files for developer assistance in the future.